tests/fixtures/current_user_capitalized_context_types.json

```diff
@@ -0,0 +1,152 @@
+{
```

Comment:
I'm not sure that a separate file for capitalization is necessary. Appending `_cap_context_type` or something like that to the new fixture names, but in `current_user.json` is probably fine.

@jessemcbride thoughts?

Reply:
Agreed.